contributing/using-git.txt

```diff
@@ -842,6 +842,13 @@ Pull Request.
     git push gh new_stuff
     # Now go to the new_stuff branch on github.com and open the PR
 
+GitHub's "Open a pull request" page invites you to leave a comment under
+the PR title. A good choice of PR title and description are both helpful
```

Comment:
I wonder if it is worth mentioning something about GitHub creating a title from the branch name (or commit message if the PR is just one commit) but that a more helpful title might be considered?